packages/date-time/src/components/Calendar/docs/CalendarDos.md

```diff
@@ -1,5 +1,5 @@
-- Use the control as a single entity.
-- Set the default date to the current date unless a specific date is required for context (e.g. the date of the conference).
-- The control is designed to resize relative to available screen width. Allow it to render in either wide or narrow as appropriate.
-- When the control is engaged, the Calendar renders as a flyout and has defined widths (300px -narrow and 440px – wide). Plan your UI implementation accordingly.
-- The control renders date in a specific format. If allowing for manual entry of date, provide helper text in the appropriate format.
+- Do use the control as a single entity.
```

Comment:
I'm not sure we ought to be making a systematic change to the wording like this without consulting our writers or at least the website designers. I suspect they'd be against prefixing everything with "Do" in particular because it adds verbosity and 

Also, all of the dos/don'ts will be going away at some point in the future per the content team, though I don't remember the details of the reasoning (@micahgodbolt do you remember?).

Reply:
Yes, dos/don'ts are going away. The question is what next steps do we take?

1. don't do anything
2. try to improve them on our own (this PR)
3. work with content team to improve what we have
4. work with content team to roll out their updated content

Reply:
I'm okay with trying to improve them on our own, just hesitant to make such a dramatic all-up stylistic change (prepending do/don't everywhere) without consulting anyone.

Reply:
I remember when writing the dos and donts for Shimmer and HoverCard someone in design team told me there is no need to add the Do and Don't into each bullet point if we already sort them in two columns with DO and DONT headers and add to that the icons on each statement has a good visual indicator if this is a good or bad thing. Adding this all over it's just redundant.

Reply:
I strongly recommend adding dos and don'ts prefixes, even if it's verbose. When you read them, it's super unclear what's being said without doing mental math that it's being said in the context of a Do or a Don't.

Reply:
The fact that we as devs have strong opposing opinions about this reinforces what I said about getting a writer's input before making such a dramatic change.

Reply:
Let me clarify; 

strongly recommend the "don't" prefixes.

I'm fine limiting the scope of the change.

Reply:
+1 to what @ecraig12345 said. I only put in there what a designer gave me. I don't argue that the example with Shimmer you brought is very confusing but it's a bit less confusing when you look at it from with context
![image](https://user-images.githubusercontent.com/29514833/82716855-89b5da00-9c4e-11ea-8804-1152d56bfcb3.png)

User will get the message right away when he visually scans and sees the header and the icon.

In general I don't argue that it needs to be changed but a consulting with design + content writer is a must in my opinion when doing it like this in bulk.

Reply:
Blind users will be read those out-loud. Just another consideration.

Reply:
The do's were removed. I think leaving the dont's makes it more clear when listening via screen reader and create some important distinction to keep things clear.